tsconfig.json

```diff
@@ -17,7 +17,6 @@
     "noImplicitReturns": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
Why remove this?

Reply:
https://medium.com/@david.zhao.blog/fix-option-suppressimplicitanyindexerrors-is-deprecated-and-will-stop-functioning-in-typescript-cee57e142028